db/app/controllers/comments.c.js

```diff
@@ -23,14 +24,15 @@ exports.getCommentByReviewId = (req, res, next) => {
 
 exports.postCommentByReviewId = (req, res, next) => {
   const { review_id } = req.params;
+  const { username } = req.body;
   const allPromises = Promise.all([
     selectReviewById(review_id),
+    checkUsernameById(review_id, username),
```

Comment:
this `checkUsernameById` check isn't strictly necassary - the `insertCommentByReviewId` model would have thrown an error if the username didn't exist already